pg_mon.c

```diff
@@ -219,6 +219,18 @@ _PG_init(void)
         if (!process_shared_preload_libraries_in_progress)
 		    return;
 
+        DefineCustomIntVariable("pg_mon.max",
```

Comment:
I see pg_stat_statements also just calls it `max` so good to go.

But honestly `max` does not help. This could be max milliseconds or max anything.

`number_of_statements = 5000`